src/main/java/com/microsoft/graph/models/extensions/Application.java

```diff
@@ -329,6 +338,22 @@ public void setRawObject(final ISerializer serializer, final JsonObject json) {
             extensionProperties = new ExtensionPropertyCollectionPage(response, null);
         }
 
+        if (json.has("homeRealmDiscoveryPolicies")) {
+            final HomeRealmDiscoveryPolicyCollectionResponse response = new HomeRealmDiscoveryPolicyCollectionResponse();
+            if (json.has("homeRealmDiscoveryPolicies@odata.nextLink")) {
```

Comment:
Can you extract this word to constant and give more meaningful name?

Reply:
Making an array of constants would be helpful here to remove some of the repeated string allocations. More meaningful names.... we can't do that as these are generated from metadata files, and we don't get to review every bit of metadata.